haiopy/buffers.py

```diff
@@ -0,0 +1,218 @@
+import numpy as np
+import pyfar as pf
+from abc import abstractproperty, abstractmethod
+
+
+class Buffer(object):
```

Comment:
What about declaring it private? similar to _Audio in the pyfar package
```suggestion
class _Buffer(object):
```